document-warehouse/get-document.js

```diff
@@ -0,0 +1,69 @@
+/** Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+async function main(
+  projectNumber = 'YOUR_PROJECT_NUMBER',
+  location = 'YOUR_PROJECT_LOCATION',
+  documentId = 'YOUR_DOCUMENT_ID',
+  userId = 'user:xxx@example.com'
+) {
+  // [START contentwarehouse_get_document]
+
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   * const projectNumber = 'YOUR_PROJECT_NUMBER';
+   * const location = 'YOUR_PROJECT_LOCATION'; // Format is 'us' or 'eu'
+   * documentId = 'YOUR_DOCUMENT_ID';
+   * const userId = 'user:xxx@example.com'; // Format is "user:xxx@example.com"
+   */
+
+  // Import from google cloud
+  const {DocumentServiceClient} = require('@google-cloud/contentwarehouse').v1;
+
+  // Create service client
+  const serviceClient = new DocumentServiceClient();
+
+  // Get Document Schema
+  async function getDocument() {
+    // Initialize request argument(s)
+    const documentRequest = {};
+
+    // The full resource name of the location, e.g.:
+    // projects/{project_number}/locations/{location}/documents/{document_id}
+    const name = `projects/${projectNumber}/locations/${location}/documents/${documentId}`;
+    documentRequest.name = name;
+
+    // Metadata Definition
+    documentRequest.requestMetadata = {userInfo: {id: userId}};
```

Comment:
Load all of these field at object creation, and use the built-in classes whenever possible.

Reply:
Perhaps, I'm looking in the wrong place. But, I don't see a built-in class for document path listed in this doc: https://cloud.google.com/nodejs/docs/reference/contentwarehouse/latest/contentwarehouse/v1.documentserviceclient#_google_cloud_contentwarehouse_v1_DocumentServiceClient_documentLinkPath_member_1_

Reply:
Fixed the field at object creation request